terraform/modules/rds_migration/logging.tf

```diff
@@ -0,0 +1,12 @@
+# TODO CLDC-2542 remove/modify this temporary logging setup
+#tfsec:ignore:aws-cloudwatch-log-group-customer-key:TODO CLDC-2542 create and encrypt with a KMS key if retaining this log group
+resource "aws_cloudwatch_log_group" "this" {

```

Comment:
FYI @SamSeed-Softwire - in case it impacts anything on [CLDC-2779](https://dluhcdigital.atlassian.net/browse/CLDC-2779) (improve cloudwatch log groups)